pom.xml

```diff
@@ -14,41 +14,54 @@
         <maven.compiler.release>17</maven.compiler.release>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
-        <spring-boot.version>2.7.5</spring-boot.version>
-        <jackson-databind-nullable.version>0.2.2</jackson-databind-nullable.version>
-        <org.mapstruct.version>1.4.2.Final</org.mapstruct.version>
-        <org.projectlombok.version>1.18.20</org.projectlombok.version>
-        <org.projectlombok.e2e-checks.version>1.18.20</org.projectlombok.e2e-checks.version>
+        <frontend-generated-sources-directory>..//kafka-ui-react-app/src/generated-sources
+        </frontend-generated-sources-directory>
+        <sonar.organization>provectus</sonar.organization>
+        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
         <git.revision>latest</git.revision>
-        <kafka-clients.version>3.2.0</kafka-clients.version>
+
+        <!-- Dependency versions -->
+        <antlr4-maven-plugin.version>4.7.1</antlr4-maven-plugin.version>
+        <apache.commons.version>2.11.1</apache.commons.version>
+        <assertj.version>3.19.0</assertj.version>
+        <avro.version>1.11.1</avro.version>
+        <byte-buddy.version>1.12.18</byte-buddy.version>
+        <confluent.version>7.3.0</confluent.version>
+        <datasketches-java.version>3.1.0</datasketches-java.version>
+        <groovy.version>3.0.13</groovy.version>
+        <jackson-databind-nullable.version>0.2.4</jackson-databind-nullable.version>
+        <jackson.version>2.14.0</jackson.version>
+        <kafka-clients.version>3.3.1</kafka-clients.version>
+        <netty.version>4.1.85.Final</netty.version>
+        <org.mapstruct.version>1.4.2.Final</org.mapstruct.version>
+        <org.projectlombok.version>1.18.24</org.projectlombok.version>
+        <protobuf-java.version>3.21.9</protobuf-java.version>
+        <reactor-netty.version>1.1.0</reactor-netty.version>
+        <scala-lang.library.version>2.13.9</scala-lang.library.version>
```

Comment:
@joschi any idea how did this thing get in here? There's no scala library prior to your commit 🤔

Reply:
@Haarolean Kafka (`org.apache.kafka:kafka_2.13` used in `kafka-ui-e2e-checks`) is written in Scala and was pulling in a vulnerable version.

This property is only used in dependency management, so that Maven will pull in the non-vulnerable version of the Scala standard library instead.

https://github.com/personio/kafka-ui/blob/2104d81e139bb6b489594dc10caf676ea725dcf9/pom.xml#L132-L136

Reply:
@joschi thank you very much, I prefer to indicate this in poms, I'll add the comment